flink-ml-lib/src/test/java/org/apache/flink/ml/recommendation/SwingTest.java

```diff
@@ -220,11 +224,23 @@ public void testTransform() {
 
     @Test
     public void testSaveLoadAndTransform() throws Exception {
-        Swing swing = new Swing().setMinUserBehavior(1);
+        Swing swing = new Swing().setMinUserBehavior(2).setMaxUserBehavior(3);
         Swing loadedSwing =
                 TestUtils.saveAndReload(tEnv, swing, tempFolder.newFolder().getAbsolutePath());
         Table outputTable = loadedSwing.transform(inputTable)[0];
         List<Row> results = IteratorUtils.toList(outputTable.execute().collect());
         compareResultAndExpected(results);
     }
+
+    @Test
+    public void testSamplingMethod() {
```

Comment:
Just wondering the purpose of this test is to make sure different seed gives different results?  It seems not quite meaningful.